aikau/src/main/resources/alfresco/forms/controls/css/MultiSelect.css

```diff
@@ -0,0 +1,174 @@
+.alfresco-forms-controls-MultiSelect {
+   border: 1px solid #666;
+   box-shadow: inset 0 0 2px rgba(0,0,0,.3);
+   box-sizing: border-box;
+   padding: 5px 5px 0;
+   position: relative;
+   * {
+      box-sizing: inherit;
+   }
+   &__container {
+   }
+   &__choice {
+      background: #eee linear-gradient(to bottom, #fff, #eee);
+      border: 1px solid #666;
+      border-radius: 3px;
+      cursor: default;
+      display: inline-block;
+      margin: 0 5px 5px 0;
+      padding: 0 20px 0 3px;
+      position: relative;
+      &__content {
+         color: #333;
+         line-height: 22px;
+      }
+      &__close-button {
+         color: #aaa;
+         cursor: pointer;
+         display: block;
+         font-family: @bold-font;
+         font-size: 16px;
+         font-weight: 700;
```

Comment:
Not sure if we want to use weight, here... if looking to make something bolder then we should use the bold font variable (which is a different font). We have tried to avoid setting weight at all.